data/create-tables.js

```diff
@@ -0,0 +1,39 @@
+const client = require('../lib/client');
+
+// async/await needs to run in a function
+run();
+
+async function run() {
+
+    try {
+        // initiate connecting to db
+        await client.connect();
+    
+        // run a query to create tables
+        await client.query(`
+            CREATE TABLE users (
+                id SERIAL PRIMARY KEY,
+                email VARCHAR(256) NOT NULL,
+                hash VARCHAR(512) NOT NULL,
+                shown_name VARCHAR(256) NOT NULL
+            );
+
+            CREATE TABLE todos (
+                id SERIAL PRIMARY KEY NOT NULL,
+                task VARCHAR(512) NOT NULL,
+                complete BOOLEAN NOT NULL DEFAULT FALSE
```

Comment:
you need to create relationship between the two tables and connect todos to specific users.

                user_id INTEGER NOT NULL REFERENCES users(id)